PocketCastsTests/Tests/Player/Transcripts/ComposeFilterTests.swift

```diff
@@ -0,0 +1,71 @@
+import XCTest
+
+@testable import podcasts
+
+final class ComposeFilterTests: XCTestCase {
```

Comment:
I added some tests and I believe we're still quite not there yet. For the transcripts using the `<v person>` the content is not correctly formatted in sentences, it just becomes a huge bulk of text:

<img src="https://github.com/Automattic/pocket-casts-ios/assets/7040243/1ab8d690-9c8a-4477-be58-3f866e67666f" width="300">